sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Integration.Tests/Azure.Monitor.OpenTelemetry.Exporter.Integration.Tests.csproj

```diff
@@ -31,9 +32,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\applicationinsights\Microsoft.Azure.ApplicationInsights.Query\src\Microsoft.Azure.ApplicationInsights.Query.csproj" />
     <ProjectReference Include="..\..\..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj" />
```

Comment:
Is there a way to use something like `$(AzureCoreSharedSources)`? Ideally we might want to avoid all the relative paths.

Reply:
I found a variable for the Azure.Core.TestFramework and will switch to use that. 
But they don't seem to have a variable for the root sdk directory. I can open an issue to suggest one, but i wouldn't want to hold up this PR for this one issue.

https://github.com/Azure/azure-sdk-for-net/blob/a2c190be57caf4fec4c3c82be42019310ed95f2f/eng/Directory.Build.Data.props#L164-L166